File: problems/002-two-sum/rust/practice.rs
use crate::json::JVal;

pub fn solve(c: &JVal) -> JVal {
    let nums = c.get("nums").as_arr();
    let target = c.get("target").as_num();
    // Return the indices [i, j] (i < j) of the two numbers in nums that add up to target.
    JVal::Arr(vec![])
}

File: problems/002-two-sum/zig/practice.zig
const std = @import("std");
const json = std.json;

pub fn solve(alloc: std.mem.Allocator, c: json.Value) !json.Value {
    _ = alloc;
    const nums = c.object.get("nums").?.array.items;
    const target = c.object.get("target").?;
    // Return the indices [i, j] (i < j) of the two numbers in nums that add up to target.
    _ = nums;
    _ = target;
    return .{ .array = json.Array.init(std.heap.page_allocator) };
}
